diff --git a/build.gradle b/build.gradle
--- a/build.gradle
+++ b/build.gradle
@@ -40,9 +40,14 @@
 ext['tomcat.version'] = '8.5.16'
 dependencies {
+    compile 'org.springframework.boot:spring-boot-autoconfigure:1.5.4.RELEASE'
+    compile 'org.springframework:spring-beans:4.3.9.RELEASE'
+    compile 'org.springframework.boot:spring-boot:1.5.4.RELEASE'
+    compile 'org.slf4j:slf4j-api:1.7.25'
+    compile 'org.springframework:spring-context:4.3.9.RELEASE'
+    compile 'com.vaadin.external.google:android-json:0.0.20131108.vaadin1'
+    compile 'org.springframework:spring-web:4.3.9.RELEASE'
-    compile("org.springframework.boot:spring-boot-starter-web")
     compile 'org.json:json:20140107'
-    testCompile('org.springframework.boot:spring-boot-starter-test')
-    testCompile('com.jayway.jsonpath:json-path')
+    testCompile 'com.jayway.jsonpath:json-path'
 }
 
 task wrapper(type: Wrapper) {
